docs/examples/datasets/bekendeAmsterdammers/dataset.json

```diff
@@ -4,6 +4,13 @@
   "title": "Bekende Amsterdammers",
   "description": "Bekende historische personen die in Amsterdam hebben gewoond.\n (Een voorbeeld dataset)",
   "status": "beschikbaar",
+  "owner": "Gemeente Amsterdam",
+  "publisher": "datapunt@amsterdam.nl",
+  "contactPoint": {
+    "name": "myDataTeam",
+    "email": "myDataTeam@amsterdam.nl"
+  },
+  "authorizationGrantor": "myDataTeam",
```

Comment:
Dit bestand wordt verwerkt in de documentatie. Daar wordt regel x tot en met y van het bestand opgenomen. Als je hier regels toevoegd moet de documentatie daardoor ook aangepast.

Reply:
Anders staan de verkeerde regels in de documentatie

Reply:
`ams-schema-spec.bs` heeft een n+=7 gehad voor embedded bekendeAmsterdammers example in f5b3ab5f0e5ff08577314cb5b91d5af4e3eb330d